model/Human-GEM.yml

```diff
@@ -242479,18 +242592,19 @@
       - metabolites: !!omap
           - MAM01285c: 1
           - MAM01371c: -1
+          - MAM01597c: 1
           - MAM02039c: 1
-          - MAM02040c: -1
-          - MAM02747c: -1
-          - MAM02747x: 1
+          - MAM02040c: -2
```

Comment:
Same problem as above comments.

Reply:
ah yea it was supposed to be `MAM02040c`; I'm not sure why I had `MAM02040m` in the issue and PR, but I've updated both of those to match what I actually wrote in `model/Human-GEM.yml`